paths is not works on windows

There is missing slashes in path
```
Module not found: Error: Can't resolve 'C:UsersUserDocumentsProjectsClientsJSClientAdminPortalsrc/app/Apps/Wizard/AppSuggestions/Components/AppSuggestion/AppSuggestionComponent.html'
```

--- a/index.js
+++ b/index.js
@@ -11,6 +11,6 @@ module.exports = function(source) {
   }
 
   return source.replace(templateUrlRegex, function(match, url, ending) {
-    return 'template: require("' + path.posix.join(basePath, url) + '")' + ending;
+    return 'template: require(' + loaderUtils.stringifyRequest(this, path.posix.join(basePath, url)) + ')' + ending;
   });
 };

